content/player/blockchain-integration/transactions-in-polygon.md

```diff
@@ -20,23 +20,31 @@ As stated in the the official Polygon website, [Polygon](https://polygon.technol
 MATIC is Polygon’s native token. It is a cryptocurrency used to cover gas fees in the Polygon network, among other use cases. MATIC is to Polygon as ETH is to Ethereum.
 You can buy MATIC in most cryptocurrency exchanges.
 
-## Polygon in Decentraland’s Marketplace
+## Polygon in Decentraland’s Marketplace and Builder
 
-By using the Polygon network, and thanks to Decentraland’s DAO, users can list, sell and buy wearables for free. But **transactions in Polygon are not free**. The Decentraland Foundation covers the cost of the transactions in Polygon, but there is a “free transaction limit”.
+By using the Polygon network, and thanks to Decentraland’s DAO, users can list, sell and buy wearables in the Marketplace or publish their collections in the Builder without paying for the transaction gas. 
```

Comment:
Do you think that it is worth mentioning `meta-transactions`?

Reply:
I've changed it to include `using the meta-transactions services`, would you mind checking it out?